### DISCUSSION TITLE
Any ideas about suport the MSSQL and Oracle database ?

### DISCUSSION THREAD

**Discussion by Shaun-Avaya** [diesel-rs/diesel#3027] [Q&A]
Do we hava a plan to build a new feature to suppor the more types of database (Mssql or Oracle )?

**Comment by weiznich** [comment]
We do not plan to add support for additional databases to diesel itself, as diesel is designed to support out of tree connection implementations as well. For oracle support there is already an third party crate on github, for mssql we wait for a motivated person to step up to write such an crate.

**Reply by Shaun-Avaya** [reply]
Ok, got it...
